tests/expr_and_series/unary_test.py

```diff
@@ -20,7 +16,8 @@ def test_unary(constructor: Constructor, request: pytest.FixtureRequest) -> None
             z_min=nw.col("z").min(),
             z_max=nw.col("z").max(),
         )
-        .select(nw.col("a_mean", "a_sum", "b_nunique", "z_min", "z_max").unique())
+        .unique(["a_mean", "a_sum", "b_nunique", "z_min", "z_max"])
+        .select(["a_mean", "a_sum", "b_nunique", "z_min", "z_max"])
```

Comment:
Can't we select directly? Not sure what the unique is for since these are aggregation:

```py
    result = (
        nw.from_native(constructor(data))
        .select(
            a_mean=nw.col("a").mean(),
            a_sum=nw.col("a").sum(),
            b_nunique=nw.col("b").n_unique(),
            z_min=nw.col("z").min(),
            z_max=nw.col("z").max(),
        )
    )

Reply:
you're right, thanks
```diff
--- a/tests/expr_and_series/unary_test.py
+++ b/tests/expr_and_series/unary_test.py
@@ -9,15 +9,13 @@ def test_unary(constructor: Constructor) -> None:
     data = {"a": [1, 3, 2], "b": [4, 4, 6], "z": [7.0, 8, 9]}
     result = (
         nw.from_native(constructor(data))
-        .with_columns(
+        .select(
             a_mean=nw.col("a").mean(),
             a_sum=nw.col("a").sum(),
             b_nunique=nw.col("b").n_unique(),
             z_min=nw.col("z").min(),
             z_max=nw.col("z").max(),
         )
-        .unique(["a_mean", "a_sum", "b_nunique", "z_min", "z_max"])
-        .select(["a_mean", "a_sum", "b_nunique", "z_min", "z_max"])
```
would also have worked